### DISCUSSION TITLE
RBAC told to work, yet is "coming soon" on current release

### DISCUSSION THREAD

**Discussion by Bokitza** [nocodb/nocodb#231] [Discord Threads]
is it on the road map? is it a commercial teaser?

**Comment by o1lab** [comment]
We've RBAC at couple of levels, which one was this ? at project metadata level or spreasheet view level

**Reply by Bokitza** [reply]
Spreadsheet view. Want to generally granuralize user view by role.

No option to add roles/edit rbac
